app/views/user_mailer/added_to_collection_notification.html.erb

```diff
@@ -1,19 +1,14 @@
 <% content_for :message do %>
-  <p>Dear <%= style_bold(@user.default_pseud.byline) %>,</p>
+  <p><%= t(".greeting", name: style_bold(@user.default_pseud.byline)).html_safe %></p>
 
-  <p>The collection maintainers of <%= style_link(@collection.title, collection_url(@collection)) %> have
-    added your work (<%= style_link(@work.title.html_safe, work_url(@work)) %>) to their collection!</p>
+  <p><%= t(".part1", collection: style_link(@collection.title, collection_url(@collection)).html_safe, work: style_link(@work.title.html_safe, work_url(@work)).html_safe).html_safe %></p>
 
-  <p>You are receiving this notification because you have previously elected to allow
-  automatic inclusion of your works into collections on the Archive of Our Own.</p>
+  <p><%= t(".part2") %></p>
 
-  <p>If you would like to remove your work from this collection, please visit your
-   <%= style_link("Approved Collection Items page", user_collection_items_url(@user) + "?approved=true") %>.</p>
+  <p><%= t(".html.part3", link: style_link(t(".html.link_text").html_safe, user_collection_items_url(@user) + "?approved=true").html_safe).html_safe %></p>
```

Comment:
There are multiple links in this email, so we should probably be more specific with our names. It looks like `approved_items` in place of `link` and `approved_items_text` in place of `link_text` might be consistent with the approach in #3889? 

(If we go with that, changing line 10 to use `style_link(t(".html.preferences_text")` would keep things consistent within the file.)

Reply:
Done